### PR TITLE
activeDatetime default to authorDatetime if active_datetime is not provided

### PR DESCRIPTION
**Pull request by hossenlopp**
Pull requests into CQL QDM Patient API require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1504 https://jira.mitre.org/browse/BONNIE-1512
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:

| branch | coverage |
| ------ | ---------- |
| master | Statements   : 74.25% ( 1629/2194 )<br>Branches     : 53.76% ( 415/772 )<br>Functions    : 64.96% ( 317/488 )<br>Lines        : 72.35% ( 1405/1942 ) |
| this | Statements   : 74.28% ( 1629/2193 )<br>Branches     : 53.88% ( 417/774 )<br>Functions    : 65.09% ( 317/487 )<br>Lines        : 72.39% ( 1405/1941 ) |

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests: N/A
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests: N/A
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests: N/A
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: app/assets/javascripts/datatypes/immunization.js.coffee [hunk 1]
```diff
@@ -86,8 +86,14 @@ class CQL_QDM.ImmunizationOrder extends CQL_QDM.QDMDatatype
   ###
   constructor: (@entry) ->
     super @entry
-    @_activeDatetime = CQL_QDM.Helpers.convertDateTime(@entry.active_datetime)
     @_authorDatetime = CQL_QDM.Helpers.convertDateTime(@entry.start_time)
+
+    # if activeDatetime is provided, use it, otherwise default to authorDatetime per QDM spec
+    if @entry.active_datetime?
```
Comment (mayerm94) [inline app/assets/javascripts/datatypes/immunization.js.coffee]: Model info file has activeDateTime in medicatiionOrder as well, this logic should be applied to the constructor of the medicationOrder datatype

Comment (mayerm94) [reply app/assets/javascripts/datatypes/immunization.js.coffee]: Disregard comment above. Aligning to QDM Spec, not Model info.

**Comment by mayerm94** [comment]
Should remove active datetime support from medication order
